src/app/components/applications/SSV/MyAccount/components/ClusterDashboard/ClusterDashboard.tsx

```diff
@@ -41,7 +41,7 @@ const ClusterDashboard = () => {
   const [loadingCluster, setLoadingClusters] = useState(false);
   const [loadingFeeRecipient, setLoadingFeeRecipient] = useState(false);
   const { page, pages, per_page, total } = myAccountStore.ownerAddressClustersPagination;
-  const { getNextNavigation } = validatorRegistrationFlow(location.pathname);
+  const { getNextNavigation } = validatorRegistrationFlowHook(location.pathname);
```

Comment:
somehow the name is not **useValidatorRegistrationFlow**

Reply:
yeah something didn't work in the refactor. fixed manuly.